src/Microsoft.Android.Runtime.NativeAOT/Android.Runtime.NativeAOT/NativeAotTypeManager.cs

```diff
@@ -3,33 +3,32 @@
 using Java.Interop;
 using Java.Interop.Tools.TypeNameMappings;
 
-namespace NativeAOT;
+namespace Microsoft.Android.Runtime;
 
 partial class NativeAotTypeManager : JniRuntime.JniTypeManager {
 
 	const DynamicallyAccessedMemberTypes Constructors = DynamicallyAccessedMemberTypes.PublicConstructors | DynamicallyAccessedMemberTypes.NonPublicConstructors;
 	internal const DynamicallyAccessedMemberTypes Methods = DynamicallyAccessedMemberTypes.PublicMethods | DynamicallyAccessedMemberTypes.NonPublicMethods;
 	internal const DynamicallyAccessedMemberTypes MethodsAndPrivateNested = Methods | DynamicallyAccessedMemberTypes.NonPublicNestedTypes;
 
-	// TODO: list of types specific to this application
-	Dictionary<string, Type> typeMappings = new () {
-		["android/app/Activity"]                = typeof (Android.App.Activity),
-		["android/app/Application"]             = typeof (Android.App.Application),
-		["android/content/Context"]             = typeof (Android.Content.Context),
-		["android/content/ContextWrapper"]      = typeof (Android.Content.ContextWrapper),
-		["android/os/BaseBundle"]               = typeof (Android.OS.BaseBundle),
-		["android/os/Bundle"]                   = typeof (Android.OS.Bundle),
-		["android/view/ContextThemeWrapper"]    = typeof (Android.Views.ContextThemeWrapper),
-		["my/MainActivity"]                     = typeof (MainActivity),
-		["my/MainApplication"]                  = typeof (MainApplication),
-	};
+	static readonly IDictionary<string, Type> TypeMappings = new Dictionary<string, Type> (StringComparer.Ordinal);
 
 	public NativeAotTypeManager ()
 	{
 		AndroidLog.Print (AndroidLogLevel.Info, "NativeAotTypeManager", $"# jonp: NativeAotTypeManager()");
+		InitializeTypeMappings ();
```

Comment:
This likewise should be in the static constructor.

Alternatively, `TypeMappings` should be an *instance* field.

Reply:
I made it an instance field, as I think it makes more sense for the instance of the `*TypeManager` to hold its typemap.